examples/basics/custom-type/custom-type.go

```diff
@@ -25,17 +29,24 @@ type (
 )
 
 func main() {
-	// Connect to a local Qlik Associative Engine
+	// Fetch the QCS_HOST and QCS_API_KEY from the environment variables
+	QCS_HOST := os.Getenv("QCS_HOST")
+	QCS_API_KEY := os.Getenv("QCS_API_KEY")
+
+	// Connect to Qlik Cloud tenant
 	ctx := context.Background()
-	global, err := enigma.Dialer{}.Dial(ctx, "ws://localhost:9076", nil)
+	rand.Seed(time.Now().UnixNano())
+	global, err := enigma.Dialer{}.Dial(ctx, fmt.Sprintf("wss://%s/app/SessionApp_%v", QCS_HOST, rand.Int()), http.Header{
```

Comment:
Could be good with a comment regarding the `/SessionApp_123` thing. I assume this how session apps are created in QCS?

Reply:
added here: https://github.com/qlik-oss/enigma-go/pull/222/files#diff-49aaa2819e35a856818ecec8c9fa7e1c79ad028d3f44bd749736353cfb51bac9R10